common/views.py

```
@@ -92,6 +93,47 @@ def user_login(request_data, model_class, serializer_class, user_type_name='user
         except AttributeError: return Response({'error': f'You are not registered as a {user_type_name}.'}, status=status.HTTP_400_BAD_REQUEST)
     else: return Response({'error': 'Invalid credentials.'}, status=status.HTTP_401_UNAUTHORIZED)
 
+def verify_user(request, email, model_class, user_type='user'):
+    otp = request.data.get('otp')
+    if request.META.get('HTTP_X_FORWARDED_FOR'): ip = request.META.get('HTTP_X_FORWARDED_FOR').split(',')[0]
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)